reservacionVuelo/App.js

```diff
@@ -10,11 +10,9 @@ class App extends Component {
   }
 
   render() {
-    return (
-      <View>
-        <Text> {this.props.name} </Text>
-      </View>
-    );
+    return (  
+      <SignUp> </SignUp>
+    );  
```

Comment:
Same about indentation and ESLint, comments about that I left in SignUp.js file.

And WDYT about instead of "`<SignUp> </SignUp>`" having "`<SignUp />`"?
Although it isn't necessary, whatever you prefer and works best for you c: